src/gui/touchscreengui.h

```diff
@@ -318,6 +330,9 @@ class TouchScreenGUI
 
 	// rare controls bar
 	AutoHideButtonBar m_rarecontrolsbar;
+
+	// device active state
+	static bool m_active;
```

Comment:
`m_` used only for private variables.

Reply:
Is there any reason why this variable is static?

Reply:
Because there is no touchscreen gui in main menu. I have to store somewhere information that user clicked "start game" using mouse, so that the touch gui will be already hidden at the beginning of game. 

Maybe it would be better to store it in input handler but there is easy access to touch gui everywhere and the touch gui is global variable anyway.